epochStart/bootstrap/syncEpochStartMeta.go

```diff
@@ -155,6 +155,11 @@ func (e *epochStartMetaSyncer) initTopicForEpochStartMetaBlockInterceptor() erro
 		return err
 	}
 
+	err = e.messenger.UnregisterMessageProcessor(factory.MetachainBlocksTopic)
```

Comment:
This is sufficient for the scope of this PR?

Reply:
forgot to push.